### PR TITLE
Safer YaST countdown for svirt

### PR DESCRIPTION
**Pull request by mnowaksuse**
After #6919 safer YaST countdown is disabled on svirt. Though, the approach should work everywhere but "HW-based backends".

Validation runs:
* `default_install@svirt-vmware`: https://openqa.suse.de/tests/2518756
* `textmode@svirt-vmware`: https://openqa.suse.de/tests/2518755

### CODE REVIEW

File: tests/installation/reboot_after_installation.pm [hunk 1]
```diff
@@ -27,7 +28,7 @@ sub run {
         $svirt->change_domain_element(os => boot => {dev => 'hd'});
     }
     # Reboot
-    if (is_remote_backend) {
+    if (is_remote_backend && !is_svirt_except_s390x) {
```
Comment (okurz) [inline tests/installation/reboot_after_installation.pm]: hm, it get's the job done but I wonder if we can not find a better way to describe it to work with a single condition that is obvious to everyone. So what makes s390x in common with e.g. ipmi and spvm but not as generic as `is_remote_backend`? Can we find a proper method name to explain what we *actually* need here? Do you have an idea?

Comment (alice-suse) [reply tests/installation/reboot_after_installation.pm]: Or does this only apply to physical machines?

Comment (okurz) [reply tests/installation/reboot_after_installation.pm]: hm, I guess in principle this if should apply for all backends where the console used for installation is provided by the SUT directly and not the hypervisor. Sounds more like we want to use `has_ttys` from Utils::Backends here. @mnowaksuse does this make sense?

Comment (mnowaksuse) [reply tests/installation/reboot_after_installation.pm]: @okurz Thanks for pointing out `has_ttys()`, I think that's the right ingredient. And for the actual question: no, sorry, no idea.

**Comment by mnowaksuse** [comment]
Will merge tomorrow, unless further comments.

**Comment by alice-suse** [comment]
LGTM.

**Pull request merged**